src/sync/sensor/queries/sensors.handler.ts

```diff
@@ -0,0 +1,27 @@
+import { RetrieveSensorsQuery } from './sensors.query';
+import { LedgerConnection } from '../ledger.connection';
+import { IQueryHandler, QueryHandler } from '@nestjs/cqrs';
+
+@QueryHandler(RetrieveSensorsQuery)
```

Comment:
Een query handler als PoC? Dit zouden we niet nodig hoeven hebben ... hoewel het wellicht wel prettig is als we voor de operations wel een API ontwikkelen waarin we de status van de ledger kunnen bevragen ... 🤔

Reply:
Deze had ik gemaakt voor test doeleinden ;) Kunnen we later idd weglaten.